client/van_connector_create_integration_test.go

```diff
@@ -0,0 +1,60 @@
+// +build integration
+
+// this ^ comment here is what makes test in this file only discovered when go
+// test executed with the -tags=integration flag, i.e. "$ go test -count=1 -v -tags=integration ./client"
+
+package client_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/skupperproject/skupper/test/cluster"
+	"gotest.tools/assert"
+)
+
+type VanConnectorCreateInteriorTestRunner struct {
+	cluster.ClusterTestRunnerBase
+}
+
+func (r *VanConnectorCreateInteriorTestRunner) Setup(ctx context.Context) {
+	r.Pub1Cluster.CreateNamespace()
+}
+
+func (r *VanConnectorCreateInteriorTestRunner) TearDown(ctx context.Context) {
+	r.Pub1Cluster.DeleteNamespace()
+}
+
+//these three are the only "NON" boiler plate or duplicated lines in this file
+func (r *VanConnectorCreateInteriorTestRunner) RunTests(ctx context.Context) {
```

Comment:
this function along with the comment below are the only important things in this file, and in case of non draft pr probably this file would be no more the 6 or 7 lines.